[2024-09-19 16:56:34,612] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_save_weather_data.save_temperature manual__2024-09-19T16:49:38.497762+00:00 [queued]>
[2024-09-19 16:56:34,615] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: fetch_save_weather_data.save_temperature manual__2024-09-19T16:49:38.497762+00:00 [queued]>
[2024-09-19 16:56:34,615] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-09-19 16:56:34,615] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-09-19 16:56:34,615] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-09-19 16:56:34,619] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): save_temperature> on 2024-09-19 16:49:38.497762+00:00
[2024-09-19 16:56:34,621] {standard_task_runner.py:52} INFO - Started process 61708 to run task
[2024-09-19 16:56:34,624] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_save_weather_data', 'save_temperature', 'manual__2024-09-19T16:49:38.497762+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/part2.py', '--cfg-path', '/var/folders/ks/w6y0t6b552bdg88w8kq5g_xc0000gn/T/tmpobzq7kxy', '--error-file', '/var/folders/ks/w6y0t6b552bdg88w8kq5g_xc0000gn/T/tmp4jv4jf5t']
[2024-09-19 16:56:34,625] {standard_task_runner.py:80} INFO - Job 28: Subtask save_temperature
[2024-09-19 16:56:39,640] {task_command.py:371} INFO - Running <TaskInstance: fetch_save_weather_data.save_temperature manual__2024-09-19T16:49:38.497762+00:00 [running]> on host MBPdeCharmaine
[2024-09-19 16:56:44,658] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_save_weather_data
AIRFLOW_CTX_TASK_ID=save_temperature
AIRFLOW_CTX_EXECUTION_DATE=2024-09-19T16:49:38.497762+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-19T16:49:38.497762+00:00
[2024-09-19 16:56:44,660] {part2.py:41} INFO - Temperature data saved to file.
[2024-09-19 16:56:44,660] {python.py:173} INFO - Done. Returned value was: None
[2024-09-19 16:56:44,662] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=fetch_save_weather_data, task_id=save_temperature, execution_date=20240919T164938, start_date=20240919T165634, end_date=20240919T165644
[2024-09-19 16:56:44,665] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-09-19 16:56:44,670] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
